pr_watch/models.py

```diff
@@ -56,11 +56,14 @@ class WatchedFork(models.Model):
     # uses internal id key
     enabled = models.BooleanField(default=True)
     # This is the old .env variable WATCH_ORGANIZATION
-    organization = models.CharField(
-        max_length=200,
+    organization = models.ForeignKey(
+        Organization,
+        on_delete=models.CASCADE,
         db_index=True,
-        help_text=("Watched GitHub organization. E.g.: open-craft. PRs against the watched fork "
-                   "made by members of this organization will trigger a sandbox build"),
+        help_text=(
+            "Watched GitHub organization. E.g.: open-craft. PRs against the watched fork "
```

Comment:
@rocioar Could you update this to reflect the conceptual changes that we are making to the logic for watching PRs? You could simplify the first part to something like:

> Organization to watch. PRs against against the watched fork ..."

Reply:
👍  done